pkg/initialize/run.go

```diff
@@ -29,22 +29,23 @@ import (
 )
 
 var (
-	policyRepoPath  = config.GetPolicyRepoPath()
-	policyBasePath  = config.GetPolicyBasePath()
-	repoURL         = config.GetPolicyRepoURL()
-	branch          = config.GetPolicyBranch()
 	errNoConnection = fmt.Errorf("could not connect to github.com")
 )
 
 const terrascanReadmeURL string = "https://raw.githubusercontent.com/accurics/terrascan/master/README.md"
 
 // Run initializes terrascan if not done already
-func Run(isScanCmd bool) error {
+func Run(isNonInitCmd bool) error {
 	zap.S().Debug("initializing terrascan")
 
+	zap.S().Debugf("rego subdir path : %s", config.GetPolicyRepoPath())
```

Comment:
with this log, we would be logging rego subdir two times

Reply:
removed